build.gradle

```diff
@@ -43,6 +52,8 @@ dependencies {
     //to explicitely declare it here. If the Tekton F8 is
     //updated check the version
     compile 'com.squareup.okio:okio:1.15.0'
+    // telemetry contributes annotations 13.0.0, so we need to declare newer version
+    compile 'org.jetbrains:annotations:16.0.1'
```

Comment:
That seems very ugly and error prone, we should not rely on those dependencies

Reply:
what alternative do you suggest? Without it, compilation fails with the following error:
```
> Task :compileJava
/Users/andredietisheim/Documents/jboss-workspaces/intellij-tekton/src/main/java/com/redhat/devtools/intellij/tektoncd/settings/SettingsConfigurable.java:24: error: cannot find symbol
    public @Nls(capitalization = Nls.Capitalization.Title) String getDisplayName() {
                ^
  symbol:   method capitalization()
  location: @interface Nls
/Users/andredietisheim/Documents/jboss-workspaces/intellij-tekton/src/main/java/com/redhat/devtools/intellij/tektoncd/settings/SettingsConfigurable.java:24: error: cannot find symbol
    public @Nls(capitalization = Nls.Capitalization.Title) String getDisplayName() {
                                    ^
  symbol:   variable Capitalization
  location: @interface Nls
Note: /Users/andredietisheim/Documents/jboss-workspaces/intellij-tekton/src/main/java/com/redhat/devtools/intellij/tektoncd/listener/TreeDoubleClickListener.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors

> Task :compileJava FAILED
```

Reply:
@jeffmaury thoughts?